hw/ip/edn/data/edn.hjson

```diff
@@ -87,7 +87,7 @@
         { bits: "0",
           desc: '''
                 When true, the CTRL can be written by software.
-                When false, this field read-only. Defaults true, write one to clear.
+                When false, this field read-only. Defaults true, write zero to clear.
```

Comment:
Good catch @cindychip, at one time it used to be a rw1c register, but then a decision was made to make all of these kinds of registers rw0c.